Graffle.FlowEventProcessor/Program.cs

```diff
@@ -37,36 +37,51 @@ static async Task Main(string[] args)
             var webhookUrl = config.GetValue<string>("WebhookUrl");
             var eventId = config.GetValue<string>("EventId");
             var verbose = config.GetValue<bool?>("Verbose") ?? false;
+            var hmacToken = config.GetValue<string>("HMACToken");
 
-            if(string.IsNullOrWhiteSpace(nodeName) || (nodeName != "MainNet" && nodeName != "TestNet")) {
+            if (string.IsNullOrWhiteSpace(nodeName) || (nodeName != "MainNet" && nodeName != "TestNet"))
+            {
                 throw new Exception("Specify FlowNode environment variable of either MainNet or TestNet.");
             }
 
-            if(string.IsNullOrWhiteSpace(webhookUrl)) {
+            if (string.IsNullOrWhiteSpace(webhookUrl))
+            {
                 throw new Exception("Specify WebhookUrl environment variable.");
             }
 
-            if(string.IsNullOrWhiteSpace(eventId)) {
+            if (string.IsNullOrWhiteSpace(eventId))
+            {
                 throw new Exception("Specify EventId environment variable.");
             }
-            
-            if(!maximumBlockScanRangeEnvVariable.HasValue || maximumBlockScanRangeEnvVariable <= 0 || maximumBlockScanRangeEnvVariable > 250){
+
+            if (!maximumBlockScanRangeEnvVariable.HasValue || maximumBlockScanRangeEnvVariable <= 0 || maximumBlockScanRangeEnvVariable > 250)
+            {
                 throw new Exception("Specify MaximumBlockScanRange environment variable between 1 and 250.");
             }
 
+            if (string.IsNullOrEmpty(hmacToken) || !TryParseBase64(hmacToken, out byte[] hmacBytes))
```

Comment:
This makes HMAC required correct? I would say check if the value exists and its Base64, but they dont have to have a Token

Reply:
Made optional in commit [feb81ed](https://github.com/Graffle/flow-event-processor/pull/2/commits/feb81ed43fe98ca7867dd54d4322ddee9ff0140e)